src/Nox.Reference.Data.World/Models/VatNumbers/VatNumberValidationResult.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Text;
+using Nox.Reference.Abstractions;
+
+namespace Nox.Reference.Data.World;
+
+internal class VatNumberValidationResult : IVatNumberValidationResult
+{
+    private readonly List<string?> _validationErrors = new();
+
+    private VatNumberValidationResult()
+    {
+    }
+
+    public static VatNumberValidationResult CreateWithoutValidation(string validationError)
+    {
+        var result = new VatNumberValidationResult();
+        if (!string.IsNullOrWhiteSpace(validationError))
+        {
+            result.AddError(validationError);
+        }
+
+        result.Status = VatValidationStatus.NotVerified;
+        return result;
+    }
+
+    public static VatNumberValidationResult CreateWithValidaton(string vatNumber, string country)
+    {
+        return CreateWithValidaton(VatValidationStatus.Valid, vatNumber, country);
```

Comment:
Why status is always valid?

Reply:
Change method name or code

Reply:
CreateWithoutValidation - validation is not carried out, result status is NotVerified (or might be Unknown)
CreateWithValidaton - If validation carries out it marks result as valid by default until error happens